src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/scheduler/RepositoryScheduler.kt

```diff
@@ -0,0 +1,22 @@
+package net.leanix.vsm.gitlab.broker.connector.scheduler
+
+import net.leanix.vsm.gitlab.broker.connector.application.InitialStateService
+import net.leanix.vsm.gitlab.broker.shared.cache.AssignmentsCache
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component
+class RepositoryScheduler(
+    private val initialStateService: InitialStateService,
+) {
+
+    private val logger = LoggerFactory.getLogger(RepositoryScheduler::class.java)
+
+    @Scheduled(cron = "\${leanix.vsm.schedule:0 0 4 * * *}")
```

Comment:
Can we have an env variable to override it an use `0 0 4 * * *` as default?

Reply:
It's a good point, let's discuss it on the daily

Reply:
looks like we won't be doing this.
marking it as resolved